spec/system/candidate_interface/entering_details/candidate_entering_personal_details_when_requiring_visa_2022_spec.rb

```diff
@@ -101,17 +92,13 @@ def and_i_can_change_nationality_to_an_eu_country_with_settled_status
     choose 'EU settled status'
     click_button t('save_and_continue')
 
-    expect(page).to have_content 'When did you enter the UK?'
-    fill_in 'Day', with: '3'
-    fill_in 'Month', with: '2'
-    fill_in 'Year', with: '2001'
-    click_button t('save_and_continue')
+    expect(page).not_to have_content 'When did you enter the UK?'
```

Comment:
Maybe this line could be dropped, as no one will have ever seen this question...?